edx_credentials_themes/templates/mitpe/credentials/programs/d670134d-3155-4b8a-b058-4618bb2b3e2c/certificate.html

```diff
@@ -0,0 +1,9 @@
+{% extends "../base.html" %}
+{% load staticfiles %}
+
+{% block top_logo %} 
+<img class="organization-logo" src="{% static 'mitpe/images/mitxpro-logo.png' %} " alt="MIT XPro logo">
```

Comment:
Set this as the default in the base template, and remove the overrides for the newer programs. There is no reason future templates should override this block unless the logo changes again.

Also, "MIT xPro"